db/migrations/versions/6277d910baa8_.py

```diff
@@ -0,0 +1,74 @@
+"""empty message
+
+Revision ID: 6277d910baa8
+Revises: 4bdc171458b2
+Create Date: 2023-07-12 11:09:23.553438
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from alembic_utils.pg_extension import PGExtension
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "6277d910baa8"
+down_revision = "4bdc171458b2"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "qa_complete",
+        sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column(
+            "application_id", postgresql.UUID(as_uuid=True), nullable=True
+        ),
+        sa.Column("user_id", sa.String(), nullable=False),
+        sa.Column(
+            "date_created",
+            sa.DateTime(),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.ForeignKeyConstraint(
+            ["application_id"],
+            ["assessment_records.application_id"],
+            name=op.f("fk_qa_complete_application_id_assessment_records"),
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_qa_complete")),
+    )
+
+    op.execute("ALTER TYPE flagstatus RENAME TO flagstatus_old")
+    op.execute(
+        "CREATE TYPE flagstatus AS ENUM ('RAISED', 'STOPPED', 'RESOLVED')"
+    )
+    op.execute(
+        "ALTER TABLE flag_update ALTER COLUMN status TYPE flagstatus USING status::text::flagstatus"
+    )
+    op.execute(
+        "ALTER TABLE assessment_flag ALTER COLUMN latest_status TYPE flagstatus USING latest_status::text::flagstatus"
+    )
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    public_pg_trgm = PGExtension(schema="public", signature="pg_trgm")
+    op.create_entity(public_pg_trgm)
+
+    op.drop_table("qa_complete")
+
+    op.execute("ALTER TYPE flagstatus RENAME TO flagstatus_old")
+    op.execute(
+        "CREATE TYPE flagstatus AS ENUM ('RAISED', 'STOPPED', 'QA_COMPLETE', 'RESOLVED')"
+    )
+    op.execute(
+        "ALTER TABLE flag_update ALTER COLUMN status TYPE flagstatus USING status::text::flagstatus"
+    )
+    op.execute(
+        "ALTER TABLE assessment_flag ALTER COLUMN latest_status TYPE flagstatus USING latest_status::text::flagstatus"
+    )
```

Comment:
```suggestion
    with op.get_context().autocommit_block():
        op.execute("ALTER TYPE flagstatus ADD VALUE 'QA_COMPLETED'")
```
Are you still working on downgrade function? Looks you haven't updated it yet

Reply:
I missed this line in the downgrade function: op.execute("DROP TYPE flagstatus_old") but have added now and should be all done 👍🏾